templates/includes/navbar.html

```diff
@@ -127,11 +127,10 @@
                     <a href="/js" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">Javascript</a>
                     <a href="/ml" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">Machine Learning</a>
                     <a href="/django" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">Django</a>
+                    <a href="/c" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">C</a>
+                    <a href="/cpp" style="color: #db1212; letter-spacing: 1px;" class="nav-item nav-link">C ++</a>
                 </div>
             </div>
-            <a href="{% url 'home-team' %}" style="font-size:20px; color: #1504fa; letter-spacing: 1px; font-weight: bold;" class="nav-item nav-link">
```

Comment:
@muhammadgalhoum why u removed this ?

Reply:
> @muhammadgalhoum why u removed this ?

I have updated my branch since the first time you asked me and I usually do an update of the branch before pushing to my branch, and for the link that I deleted because I do not have the code for this link, which is in the views.py file and also in other files, and this link is for the issue It was not my responsibility and I only put it because when doing PR once I found it in the navbar so I put it and then deleted it but the code works for me perfectly correctly
I will contact one of the mentors to solve this problem

![Capture](https://user-images.githubusercontent.com/56798376/167092652-311faa98-36da-490a-ad1f-f8ab4975d69c.PNG)